### PR TITLE
WIP - Internal B9lab language and style check, review of highlight boxes

### PR DESCRIPTION
**Pull request by CitMC**
DON'T MERGE YET

Documentation content update for Module: What is Cosmos?, Main Concepts, Running a Node, My Own Chain, all sections without content updates in Q1.

This is a general language and style check, which also includes a revision of the highlight boxes due to new featured assets being available.

### Change scope

The changes in this Pull Request include (please tick all that apply):

- [x] Small language/grammar fixes
- [x] Small content fixes 
- [ ] Addition of new content
- [ ] Sample code/command updates
- [ ] Platform fixes
- [ ] Other

### CODE REVIEW

**Comment by CitMC** [comment]
@coldice Please merge into Master branch. This is Andrew's language and style review of existing content that will not be changed by the other PRs.

File: academy/1-what-is-cosmos/blockchain-and-cosmos.md [hunk 1]
```diff
@@ -34,41 +34,41 @@ This layered model can be applied to blockchains generally.
...
+Want to look closer at the initial proposition of Bitcoin? See the original [Bitcoin Whitepaper](https://bitcoin.org/bitcoin.pdf)
 
 </HighlightBox>
 
 <ExpansionPanel title="More on PoW">
 
-Bitcoin uses PoW to achieve Byzantine Fault-Tolerance (BFT) which enables decentralized, trustless networks to function even with malfunctioning or malicious nodes present. It is best described as a cryptographic puzzle solved by a network's node called a miner. The puzzle is a task of pre-defined, arbitrary difficulty. At the current scale of the networks using PoW, the outcome is akin to a lottery with a single winning node.
+Bitcoin uses PoW to achieve Byzantine Fault-Tolerance (BFT) which enables decentralized, trustless networks to function even with malfunctioning or malicious nodes present. It is best described as a cryptographic puzzle solved by a network node called a "miner". The puzzle is a task of pre-defined, arbitrary difficulty. At the current scale of the networks using PoW, the outcome is akin to a lottery with a single winning ticket, the successful miner node.
```
Comment (xavierlepretre) [inline academy/1-what-is-cosmos/blockchain-and-cosmos.md:51]:
```suggestion
Bitcoin uses PoW to achieve Byzantine Fault-Tolerance (BFT) which enables decentralized, trustless networks to function even with malfunctioning or malicious nodes present. It is best described as a cryptographic puzzle solved by a network node called a "miner". The puzzle is a task of pre-defined, arbitrary difficulty. At the current scale of the networks using PoW, the outcome is akin to a lottery with a single winning ticket, that of the successful miner node.
```

**Review by xavierlepretre — APPROVED** [review]

File: academy/3-running-a-chain/node-api-and-cli.md [hunk 2]
```diff
@@ -205,7 +205,7 @@ Just in case you need a "prettier" version:
 
 </ExpansionPanel>
 
-You can find your `chain_id` in your output, which in our build happens to be called `test-chain-rT4wZY`. Make a note of the one you have as you will need it later to determine the chain ID by passing it to `simapp` via the flag `--chain-id`.
+You can find your `chain_id` in your output, which in our build happens to be called `test-chain-rT4wZY`. Make a note of your output's name, as you will need it later to determine the chain ID by passing it to `simapp` via the flag `--chain-id`.
```
Comment (xavierlepretre) [inline academy/3-running-a-chain/node-api-and-cli.md:208]:
```suggestion
You can find your `chain_id` in your output, which in this build happens to be called `test-chain-rT4wZY`. Make a note of your output's name, as you will need it later to determine the chain ID by passing it to `simapp` via the flag `--chain-id`.
```

File: academy/3-running-a-chain/node-api-and-cli.md [hunk 3]
```diff
@@ -403,7 +403,7 @@ That is a `10` stake.
 
 ## CLI routing
 
-Now it is time for a bit of Go code. How does this `simd` interact via the command-line interface? Inspect the [`cosmos-sdk/simapp/simd/main.go`](https://github.com/cosmos/cosmos-sdk/blob/d83a3bf92c9a84bddf3f5eb6692a1101c18b42f1/simapp/simd/main.go) file:
+Now it is time for a bit of Go code. How does the `simd` interact via the command-line interface? Inspect the [`cosmos-sdk/simapp/simd/main.go`](https://github.com/cosmos/cosmos-sdk/blob/d83a3bf92c9a84bddf3f5eb6692a1101c18b42f1/simapp/simd/main.go) file:
```
Comment (xavierlepretre) [inline academy/3-running-a-chain/node-api-and-cli.md:406]:
```suggestion
Now it is time for a bit of Go code. How does `simd` interact via the command-line interface? Inspect the [`cosmos-sdk/simapp/simd/main.go`](https://github.com/cosmos/cosmos-sdk/blob/d83a3bf92c9a84bddf3f5eb6692a1101c18b42f1/simapp/simd/main.go) file:
```

**Pull request merged**